pybossa/cache/task_browse_helpers.py

```diff
@@ -256,25 +256,27 @@ def parse_tasks_browse_args(args):
                                           'created', 'finish_time', 'gold_task',
                                           'actions', 'lock_status']
     if 'display_info_columns' in args:
-        display_info_columns = json.loads(args['display_info_columns'])
+        display_info_columns = args['display_info_columns']
+
         if not isinstance(display_info_columns, list):
-            display_info_columns = []
+            display_info_columns = json.loads(display_info_columns)
```

Comment:
From code at [line 270](https://github.com/bloomberg/pybossa/pull/687/files#diff-6a0d6a373ba98daed8f88781663361f891c6bea2d82086bc293ec82b66ff0607R270), it seems display_info_columns to be an iterator/list. Should this line be instead as following, otherwise the code would break?
```python
if not isinstance(display_info_columns, list):
            display_info_columns = [json.loads(display_info_columns)]
```

Reply:
Well, we don't need to add `[..]`, because `display_info_columns` is either a list or a `json.dumps(list)`, so `json.loads()` would return a list.